src/OSWindow-SDL2/SDLOSXPlatform.class.st

```diff
@@ -56,6 +56,12 @@ SDLOSXPlatform >> allowTouchpadInertia [
 	ObjCLibrary uniqueInstance release: key.
 ]
 
+{ #category : 'configuration' }
+SDLOSXPlatform >> defaultRendererFlags [ 
+
+	^ super defaultRendererFlags | SDL_RENDERER_PRESENTVSYNC
```

Comment:
Could we add a comment?

```suggestion
	"SDL_RENDERER_PRESENTVSYNC is required on MacOS for certain screens to avoid rendering issues"
	^ super defaultRendererFlags | SDL_RENDERER_PRESENTVSYNC
```